mros1_reasoner/mros1_reasoner.rosinstall

```diff
@@ -1,15 +1,16 @@
 - git:
     local-name: mros1_reasoner
     uri: https://github.com/rosin-project/mros1_reasoner.git
-    version: mvp
+    version: virtual_env
 - git:
     local-name: metacontrol_msgs
     uri: https://github.com/rosin-project/metacontrol_msgs
 - git:
     local-name: mc_mdl_tomasys
-    uri: https://github.com/rosin-project/mc_mdl_tomasys.git
-    version: mvp
+    uri: https://github.com/tud-cor/mc_mdl_tomasys
+    version: master
 - git:
-    local-name: mc_mdl_abb
-    uri: https://github.com/rosin-project/mc_mdl_abb.git
-    version: mvp
+    local-name: catkin_virtualenv
+    uri: https://github.com/locusrobotics/catkin_virtualenv.git
+    version: 0.6.1
```

Comment:
This will become unnecessary as soon as Melodic is synced again ([ros_melodic_default.html?q=catkin_virt](http://repositories.ros.org/status_page/ros_melodic_default.html?q=catkin_virt)).

Reply:
Yes, It didn't work with the released version, so I put the latest one here. 

I've create an issue (https://github.com/tud-cor/mc_mros_reasoner/issues/88) so I don't forget to remove this once the new melodic sync happens.